### PR TITLE
add homepage with all articles

### PR DESCRIPTION
**Pull request by gwdawson**

### CODE REVIEW

File: src/components/Articles.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import '../styles/App.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMessage } from '@fortawesome/free-solid-svg-icons';
+
+export default function Articles() {
```
Comment (jadelandeg) [inline src/components/Articles.jsx:7]: why are you exporting this function?

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      setArticles(data.articles);
+    };
+    makeRequest();
+  }, []);
+
+  console.log(articles);
+
+  return (
+    <div className='Articles'>
+      {articles.map((article) => (
```
Comment (jadelandeg) [inline src/components/Articles.jsx:26]: each item in list needs a unique key

File: src/components/Articles.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import '../styles/App.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMessage } from '@fortawesome/free-solid-svg-icons';
+
+export default function Articles() {
+  const [articles, setArticles] = React.useState([]);
+
+  const backend = axios.create({
```
Comment (jadelandeg) [inline src/components/Articles.jsx:10]: will be reusing this a lot, would there be a better place to put it?

File: src/components/Articles.jsx (hunk 1 above)
Comment (gwdawson) [reply src/components/Articles.jsx:7]: Articles needs to be exported so i can access it in App.jsx sorry you cant see where its being used i accidentally pushed the to main
```js
function App() {
  return (
    <div className='App'>
      <Nav />
      <Header />
      <Routes>
        <Route path='/' element={<Articles />}></Route>
      </Routes>
    </div>
  );
}
```

File: src/utils/api.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const backend = axios.create({
+  baseURL: 'https://northcoders-news-api-v2.herokuapp.com/api',
+});
+
+export const makeRequest = async (setArticles) => {
```
Comment (karlaferry) [inline src/utils/api.js:7]: Everything looks good, however for api calls I would recommend you use function names that say what type of request you are doing. For this one, you can call it something like _getAllArticles_.

Comment (gwdawson) [reply src/utils/api.js:7]: thank you ill get that changed now!

**Review by karlaferry — CHANGES_REQUESTED** [review]
Good job. Only comment is try to make the api call function names a bit more descriptive. :) CSS looks good!

**Review by karlaferry — APPROVED** [review]
Looks good! Approved :)

**Pull request merged**